### PR TITLE
RSWEB-8033: refactor cta pattern

### PR DESCRIPTION
**Pull request by writes**
http://localhost:9000/derek/solutions/solutions-usage

### CODE REVIEW

**Comment by WidgetsBurritos** [comment]
👍

**Comment by amberRrucker** [comment]
Something is up. It should look like the second one and it def does not. Not totally sure what happened there.

screen shot 2017-01-11 at 10 48 42 am
screen shot 2017-01-11 at 10 48 50 am

**Comment by writes** [comment]
@WidgetsBurritos @amberRrucker I have updated the pattern. Let me know if you see any issues. 

Thanks!

File: styleguide/derek/_partials/solutions/cta.ejs [hunk 1]
```diff
@@ -1,21 +1,26 @@
...
+      </div>
+    </div>
+    <div class="cta-row">
+      <div class="cta-buttonWrapper">
+        <a class="cta-button">Talk to a Specialist</a>
+      </div>
+    </div>
+    <div class="cta-row">
+      <div class="cta-subheadWrapper">
+        <h3>Call <a href="javascript:void(0)">1-855-715-7307</a> to start saving</h3>
```
Comment (bighappyface) [inline styleguide/derek/_partials/solutions/cta.ejs]: Why can't this be a `p`?

Comment (writes) [inline styleguide/derek/_partials/solutions/cta.ejs]: Hey @bighappyface the update has been made.

File: styleguide/_themes/global/scss/cta.scss [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+}
+
+.cta-row {
+  @include make-row;
+}
+
+.cta-subheadWrapper {
+  @include make-md-column(12);
+
+  p {
```
Comment (bighappyface) [inline styleguide/_themes/global/scss/cta.scss]: Being that there seem to be specific styles for this text, I think it is appropriate to have a specific class for it. Such a class can be in the CTA module, but something tells me that this combination of family, size and weight may be useful elsewhere. If not, it's a snowflake.

**Comment by writes** [comment]
@WidgetsBurritos and @bighappyface would it make sense to do `div.subheadWrapper` as opposed to `.subheadWrapper p` to account for any tags put in the field or would we rather restrict to `p` and `a` with styles set on `p`? The `p` tag default styles will currently override the class alone and the subhead uses a different font-family.

File: styleguide/_themes/global/scss/cta.scss [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+}
+
+.cta-row {
+  @include make-row;
+}
+
+.cta-subheadWrapper {
+  @include make-md-column(12);
+
+  .cta-subhead {
```
Comment (bighappyface) [inline styleguide/_themes/global/scss/cta.scss]: Is this nested for greater specificity to get the CSS properties to take effect?

**Comment by amberRrucker** [comment]
Looks good. +1

File: styleguide/_themes/global/scss/cta.scss (hunk 3 above)
Comment (writes) [inline styleguide/_themes/global/scss/cta.scss]: Yes sir. It is needed to overwrite the default `p` styles.

**Comment by bighappyface** [comment]
+1

**Pull request merged**